galaxy_ng/app/api/v3/views/sync.py

```diff
@@ -47,7 +47,8 @@ def post(self, request: Request, *args, **kwargs) -> Response:
             kwargs={
                 "remote_pk": remote.pk,
                 "repository_pk": distro.repository.pk,
-                "mirror": True
+                "mirror": True,
+                "optimize": True,
```

Comment:
@bmbouter is it right ot always pass `optimize=True` as this attribute is not optional.

Reply:
the default is true, you don't need to pass it, we set it at serializer level: https://github.com/pulp/pulp_ansible/blob/master/pulp_ansible/app/serializers.py#L101

Reply:
Ok, so in our case we must provide it to enqueue the task.